homeassistant/components/sia/utils.py

```diff
@@ -26,32 +21,55 @@ def get_unavailability_interval(ping: int) -> float:
     return timedelta(minutes=ping, seconds=PING_INTERVAL_MARGIN).total_seconds()
 
 
-def get_name(port: int, account: str, zone: int, entity_type: str) -> str:
-    """Give back a entity_id and name according to the variables."""
-    if zone == HUB_ZONE:
-        return f"{port} - {account} - {'Last Heartbeat' if entity_type == DEVICE_CLASS_TIMESTAMP else 'Power'}"
-    return f"{port} - {account} - zone {zone} - {entity_type}"
-
-
-def get_entity_id(port: int, account: str, zone: int, entity_type: str) -> str:
-    """Give back a entity_id according to the variables."""
-    if zone == HUB_ZONE:
-        return f"{port}_{account}_{HUB_SENSOR_NAME if entity_type == DEVICE_CLASS_TIMESTAMP else entity_type}"
-    return f"{port}_{account}_{zone}_{entity_type}"
-
-
-def get_unique_id(entry_id: str, account: str, zone: int, domain: str) -> str:
-    """Return the unique id."""
-    return f"{entry_id}_{account}_{zone}_{domain}"
-
-
 def get_attr_from_sia_event(event: SIAEvent) -> dict[str, Any]:
     """Create the attributes dict from a SIAEvent."""
     return {
-        EVENT_ACCOUNT: event.account,
-        EVENT_ZONE: event.ri,
-        EVENT_CODE: event.code,
-        EVENT_MESSAGE: event.message,
-        EVENT_ID: event.id,
-        EVENT_TIMESTAMP: event.timestamp,
+        ATTR_ZONE: event.ri,
+        ATTR_CODE: event.code,
+        ATTR_MESSAGE: event.message,
+        ATTR_ID: event.id,
+        ATTR_TIMESTAMP: event.timestamp.isoformat(),
```

Comment:
Do we still need to convert to UTC time?

Reply:
no, it is TZ aware and in UTC.